### DISCUSSION TITLE
Error: Cannot use 'in' operator to search for 'toDate' in 1998-08-20

### DISCUSSION THREAD

**Discussion by sheissportsnetwork** [AntlerVC/firetable#912] [Support / Q&A] [answered]
👉 **Please describe how to reproduce this bug here.**

1. In Firebase I have people's birthdays setup like YYYY-MM-DD which is the format that a date type says to use
2. In Rowy, I pull in birthday as a date type
3. Rowy shows the error instead of the date, Error: Cannot use 'in' operator to search for 'toDate' in <YYYY-MM-DD>

I can show the dates as strings just fine.

**Comment by harinij** [comment, marked as answer]
@sheissportsnetwork could you share if the date value is stored in your Firestore as a `timestamp` field or as a `string`?
<img width="193" alt="Screen Shot 2022-11-02 at 2 33 35 pm" src="https://user-images.githubusercontent.com/307298/199389200-5442c68a-5e50-4710-941c-3a8e15e63a06.png">

**Reply by BeeBombshell** [reply]
@sheissportsnetwork, @gorvis I believe your dates are stored as a string in the Firestore DB, since the error statement is a type of `TypeError`.
For importing, what you could do is import the column as a **Short Text/ Long Text** field and use the derivative column with a script to convert numbers to a date format.

**Comment by gorvis** [comment]
Timestamp

**Reply by BeeBombshell** [reply]
Hey @gorvis, @sheissportsnetwork, is this still an issue?

**Reply by gorvis** [reply]
Not sure, I removed the field entirely from Rowy and I just edit these right in Firebase.
